include/Hardmon.hpp

```diff
@@ -11,14 +11,24 @@
 #include <core/utils/types/FixedQueue.hpp>
 #include <models/Hardmon_Model.hpp>
 
-namespace io = core::io;
+#include <core/rtos/Initializable.hpp>
+#include <core/rtos/Mutex.hpp>
+
+namespace io   = core::io;
+namespace rtos = core::rtos;
 
 namespace vcu {
 
 /**
- * Driver for the Hardware Monitor
+ * The Hardmon (Hardware Monitor) is one of two microcontrollers on the VCU board.
+ * It monitors the MCuC in order to determine whether or not it is operating safely and logically.
```

Comment:
Since this is the first time in this file the MCuC is mentioned

```suggestion
It monitors the MCuC (Motor Controller Microcontroller) in order to determine whether or not it is operating safely and logically.```